homeassistant/components/volvooncall.py

```diff
@@ -123,6 +126,13 @@ def __init__(self, hass, vin, attribute):
         self._state.entities[self._vin].append(self)
 
     @property
+    def entity_id(self):
```

Comment:
Also, do you still prefer that I just rename the sensors instead of providing unique `entity_id`:s? (I still prefer the latter)